### ISSUE TITLE
My server boom!

### ISSUE THREAD

**Issue by junxiesky** [jasonwynn10/VanillaEntityAI#19]
hey Brother!
Why did this plugin crash my server?
I need your help. 
```
2019-04-19 [23:08:13] [Server thread/CRITICAL]: ErrorException: "Undefined offset: 127" (EXCEPTION) in "plugins/VanillaEntityAI.phar/src/jasonwynn10/VanillaEntityAI/task/HostileSpawnTask" at line 48
2019-04-19 [23:08:13] [Server thread/DEBUG]: #0 plugins/VanillaEntityAI.phar/src/jasonwynn10/VanillaEntityAI/task/HostileSpawnTask(48): pocketmine\utils\Utils::errorExceptionHandler(integer 8, string Undefined offset: 127, string phar:///home/FtCraft/pm/plugins/VanillaEntityAI.phar/src/jasonwynn10/VanillaEntityAI/task/HostileSpawnTask.php, integer 48, array Array())
2019-04-19 [23:08:13] [Server thread/DEBUG]: #1 src/pocketmine/scheduler/TaskHandler(159): jasonwynn10\VanillaEntityAI\task\HostileSpawnTask->onRun(integer 1040)
2019-04-19 [23:08:13] [Server thread/DEBUG]: #2 src/pocketmine/scheduler/TaskScheduler(199): pocketmine\scheduler\TaskHandler->run(integer 1040)
2019-04-19 [23:08:13] [Server thread/DEBUG]: #3 src/pocketmine/plugin/PluginManager(688): pocketmine\scheduler\TaskScheduler->mainThreadHeartbeat(integer 1040)
2019-04-19 [23:08:13] [Server thread/DEBUG]: #4 src/pocketmine/Server(2559): pocketmine\plugin\PluginManager->tickSchedulers(integer 1040)
2019-04-19 [23:08:13] [Server thread/DEBUG]: #5 src/pocketmine/Server(2335): pocketmine\Server->tick()
2019-04-19 [23:08:13] [Server thread/DEBUG]: #6 src/pocketmine/Server(2196): pocketmine\Server->tickProcessor()
2019-04-19 [23:08:13] [Server thread/DEBUG]: #7 src/pocketmine/Server(1775): pocketmine\Server->start()
2019-04-19 [23:08:13] [Server thread/DEBUG]: #8 src/pocketmine/PocketMine(258): pocketmine\Server->__construct(BaseClassLoader object, pocketmine\utils\MainLogger object, string /home/FtCraft/pm/, string /home/FtCraft/pm/plugins/)
2019-04-19 [23:08:13] [Server thread/DEBUG]: #9 (1): require(string phar:///home/FtCraft/pm/PocketMine-MP.phar/src/pocketmine/PocketMine.php)
```

**Comment by junxiesky** [comment]
hey i need help！

**Comment by jasonwynn10** [comment]
Fixed in b6e6adcbf4700b28020b674a33154333669a52f2
